docs/02-app/01-building-your-application/08-deploying/01-production-checklist.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: Going to Production
+description: Before taking your Next.js application to production, here are some recommendations to ensure the best user experience.
+---
+
+Before taking your Next.js application to production, here are some recommendations to ensure the best user experience.
+
+## Overview
```

Comment:
nit, the overview doesn't really reflect the exact order of the item in the file